langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -114,6 +162,13 @@ public isolated function indexOf(string str, string substr, int startIndex = 0)
 
 # Finds the last occurrence of one string in another string.
 #
+# ```ballerina
+# "Ballerinalang is a unique programming language".lastIndexOf("lang") ⇒ 38
+#
+# // Search for the last occurrence of a string from a specific index onwards.
```

Comment:
Should we need to have an explanation in the example as we have it in the parameter description?
```suggestion
# // Search for the last occurrence of a string from a specific index backwards.
```